pkg/client/client.go

```diff
@@ -327,14 +327,14 @@ type EventStreamOptions struct {
 	Tail            int    `json:"tail,omitempty"`
 	Follow          bool   `json:"follow,omitempty"`
 	Details         bool   `json:"details,omitempty"`
-	Name            string `json:"name,omitempty"`
+	Prefix          string `json:"prefix,omitempty"`
 	ResourceVersion string `json:"resourceVersion,omitempty"`
 }
 
 func (o EventStreamOptions) ListOptions() *kclient.ListOptions {
 	fieldSet := make(fields.Set)
-	if o.Name != "" {
-		fieldSet["metadata.name"] = o.Name
+	if o.Prefix != "" {
+		fieldSet["prefix"] = o.Prefix
```

Comment:
This is a backwards incompatible change. It means a new (i.e. w/ change) client talking to an old (i.e. w/o change) API won't be able to get events by name.